### ISSUE TITLE
Reactions - Further Thread List Alignment Tweaks

### ISSUE THREAD

**Issue by lilalien** [Audentio/xf2addon-issues#197]
Running the latest version released today...

For Reference:
https://github.com/Audentio/xf2addon-issues/issues/192
https://github.com/Audentio/xf2addon-issues/issues/116

I think this could still be improved upon somewhat in the alignment of the reaction row?

On the thread list, when there are no other icons (poll, lock, sticky) the reactions still display with a bit of space to the right and aren't lined up. Any way to tweak this a tad better?

![thread_list_reactions_edit_alignment](https://user-images.githubusercontent.com/35111972/38754871-85c7d65e-3f31-11e8-98da-8bad8bb2383e.png)

**Comment by lilalien** [comment]
Its sort like, the more reactions that are visible the more right-side spacing is created...

**Comment by Nirjonadda** [comment]
Yes this happening when thread are not have close, follow or sticky icon but reactions should be showing in same line.

![38754871-85c7d65e-3f31-11e8-98da-8bad8bb2383e](https://user-images.githubusercontent.com/6899856/38755140-ebf70264-3f64-11e8-8a34-1ed0b92342bb.png)

**Comment by kylerc** [comment]
Hey guys,  Thanks for the report, looks like my fix yesterday didn't quite fix everything.  Does ```.reactions-bar.bit-list {
    transform: initial;
}

.reactions-bar.bit-list .reactions-left .th_reactions__listItem {
    transform: scale(0.75);
    margin-right: -8px;
}``` in your extra.less resolve this for you?

**Comment by lilalien** [comment]
When I toy with it, and I commend the below:

.reactions-bar.bit-list .reactions-left .th_reactions__listItem:last-of-type {
    /* margin-right: 0; */
}

...and change this one to -5px:

.reactions-bar.bit-list .reactions-left .th_reactions__listItem {
    margin-right: -5px;
}

It seems to align pretty well?

**Comment by lilalien** [comment]
Screeny of the above:

![alignment2](https://user-images.githubusercontent.com/35111972/38756382-5434420c-3f37-11e8-8183-2e72147189b8.png)

**Comment by lilalien** [comment]
Actually that might be slightly skewed, let me check something.

**Comment by lilalien** [comment]
Seems like both

.reactions-bar.bit-list .reactions-left .th_reactions__listItem {
margin-right: -5px;
}

&

.reactions-bar.bit-list .reactions-left .th_reactions__listItem:last-of-type {
    margin-right: -5px;
}

Need -5px and it seems to align??

**Comment by lilalien** [comment]
So when I try:

.reactions-bar.bit-list {
transform: initial;
}

.reactions-bar.bit-list .reactions-left .th_reactions__listItem {
transform: scale(0.75);
margin-right: -5px;
}

.reactions-bar.bit-list .reactions-left .th_reactions__listItem:last-of-type {
margin-right: -5px;
}

Seems to work nicely?

**Comment by kylerc** [comment]
Yep seems to, thanks for the help, will include this with the next update by default.

**Comment by lilalien** [comment]
Anytime! I'm an amateur... lol

...but I really enjoy tinkering! Glad to assist.
